client/scss/layouts/_page-editor.scss

```diff
@@ -622,6 +630,154 @@ footer .preview {
   }
 }
 
+#preview-panel {
+  margin-bottom: 2rem;
+
+  .preview-wrapper {
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+
+    iframe {
+      transform-origin: 0 0;
+      width: 100%;
+      display: block;
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+
+  &[data-active-preview-size='mobile'], &[data-active-preview-size].has-errors {
```

Comment:
We should avoid styles against data attributes and instead add BEM.classes as per guidelines

https://docs.wagtail.org/en/latest/contributing/ui_guidelines.html#css-guidelines

Data attributes should be for JS behaviour only.